frontend/javascripts/oxalis/workers/fetch_buffer_with_headers.worker.js

```diff
@@ -17,10 +17,13 @@ function fetchBufferWithHeaders(
       for (const [key, value] of headers.entries()) {
         headerObject[key] = value;
       }
-      return {
-        buffer,
-        headers: headerObject,
-      };
+      return transfer(
```

Comment:
Strictly speaking this change should enable the browser to not copy the data between worker and main loop, but instead move it. However, I couldn't really measure a difference (I measured the e2e loading time of opening a dataset and waiting for all buckets, but maybe there are better ways). Anyway, using transferables is more of a separate issue on its own I think.

Reply:
I opened https://github.com/scalableminds/webknossos/issues/6064.

Reply:
It's probably not immediately faster, because moving takes the same time as copying, but it might pay out in reduced GC time, instead (which is harder to measure and correlate). Just a thought, I haven't read up on it.

Reply:
I think, the main reason for `transfer` to exist is performance. However, there seems to be dependence on the size of the transferred data. [This article](https://joji.me/en-us/blog/performance-issue-of-using-massive-transferable-objects-in-web-worker/) suggests that small elements don't really benefit from being transferred (even the opposite, it can get worse). This sparked my curiosity and I did a benchmark with a dummy webworker which consumes an array buffer (size of one bucket in uint32) and returns it again. For 1000 runs, this shows:

```
no transfers: 173.39208984375 ms
only with input transfer: 112.991943359375 ms
only with output transfer: 142.373046875 ms
with all transfers: 94.93017578125 ms
```

So, the transfer seems to a have a positive effect when micro-benchmarking. So, I'd keep the change in and hope that this also has some positive effects (even when subtle).

Reply:
Sounds good! #MeasureToMakeDecisions